deploy.go

```diff
@@ -11,10 +12,10 @@ import (
 	"strings"
 	"time"
 
-	"github.com/codegangsta/cli"
 	"github.com/github/hub/git"
 	hub "github.com/github/hub/github"
-	"github.com/google/go-github/github"
+	"github.com/google/go-github/v35/github"
```

Comment:
This is the most stupid import system I think I've ever used.

Reply:
Actually though wtf, is there not a way to specify this in like the `go.mod`? It has to be in the import??

Reply:
afaict. as far as go's concerned, this is actually a different package :shrug: go modules seem like one of the worst designed package management systems i've ever seen.